plugins/modules/kea_command.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# Copyright © Thorsten Glaser <tglaser@b1-systems.de>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# note: this module has not been tested with Python, as the switch from ISC DHCPD to KEA occurs at a time where distros only ship Python3 instead.
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+---
+module: kea_command
+short_description: Submits generic command to ISC KEA server on target
+description:
+  - Submits a command to the JSON API of an ISC KEA server running on the target and obtains the result.
+  - This module supports sending arbitrary commands and returns the server response unchecked;
+    while it would be possible to write individual modules for specific KEA service commands,
+    that approach would not scale, as the FOSS hooks alone provide dozens of commands.
+  - Between sending the command and parsing the result status, RV(ignore:changed) will register as V(true) if an error occurs,
+    to err on the safe side.
+version_added: '11.3.0'
+author: Thorsten Glaser (@mirabilos)
+options:
+  command:
+    description:
+      - The name of the command to send, for example C(status-get).
+    required: true
+    type: str
+  arguments:
+    description:
+      - The arguments sent along with the command, if any.
+      - Use V({}) to send an empty JSONObject instead of omitting.
```

Comment:
Wouldn't it be better to make `default={}`? Or, leave no default and, in the code (down there somewhere):
```python
if arguments is None:
    argumentes = {}
```
Whichever makes more sense to you.

Reply:
I would not call it a `JSONObject` in the documentation for the end user, though. Some of your users might not be too familiar with JSON and its terminology. In the Ansible/YAML world, that thing would be a `dict`, even though they're amount to the same thing under the wraps.

Reply:
Ah. I was addressing the KEA user here. I guess we’ll need to fit both.

What do you two think about: `- Use V({}) to send an empty arguments dict/object instead of omitting.`